lua/indentmini/init.lua

```diff
@@ -139,11 +139,8 @@ end
 return {
   setup = function(conf)
     conf = conf or {}
-    opt.exclude = vim.tbl_extend(
-      'force',
-      { 'dashboard', 'lazy', 'help', 'markdown', 'nofile', 'terminal', 'prompt' },
-      conf.exclude or {}
-    )
+    opt.exclude = { 'dashboard', 'lazy', 'help', 'markdown', 'nofile', 'terminal', 'prompt' }
+    vim.list_extend(opt.exclude, conf.exclude or {})
```

Comment:
```suggestion
    vim.list_extend({ 'dashboard', 'lazy', 'help', 'markdown', 'nofile', 'terminal', 'prompt' }, conf.exclude or {})
```

Reply:
can be inline.

Reply:
`vim.list_extend` mutates the first list and does not return the merged list

Reply:
aha sry.